docs/TRANSLATIONS.md

```diff
@@ -1,116 +1,47 @@
+- [**Sound**](https://hosted.weblate.org/projects/organicmaps/sound/) – Text-to-speech strings for voice navigation.
+- [**Countries**](https://hosted.weblate.org/projects/organicmaps/sound/) – Country and region names.
 
-## Translation files
+## For Developers
 
-- Main:
-  - Application UI strings: [`data/strings/strings.txt`](../data/strings/strings.txt)
-  - A few iOS specific strings: [`iphone/plist.txt`](../iphone/plist.txt)
+Translations are managed by the translation team and do not require direct involvement from developers. Developers should add English base strings using the native tools provided by Android Studio and Xcode.
 
-- POI Categories:
-  - Names of map features/types: [`data/strings/types_strings.txt`](../data/strings/types_strings.txt)
-  - Search keywords/aliases/synonyms for map features: [`data/categories.txt`](../data/categories.txt)
+When implementing a new feature that already exists on another platform, please check for existing keys and reuse the same English base string to maintain consistency across platforms. Manual propagation of strings between platforms is unnecessary, as Weblate handles this automatically.
 
-  The POI definitions in the [OpenStreetMap Wiki](https://wiki.openstreetmap.org/) help finding the most suitable translation. Both POI files should be kept in sync, so make sure that every category name is also contained in the coresponding search keyword list. Strings in _categories.txt_ should, however, not contain common tokens like e.g. Shop, Store or Center as separate words.
+Regenerating strings is no longer required. Pre-filling languages using machine translation is no longer required too. Feel free to add translations for languages you know in the same PR, or you can always contribute later via Weblate.
 
-- Additional:
-  - Text-to-speech strings for navigation: [`data/strings/sound.txt`](../data/strings/sound.txt)
+Components:
 
-  - Android stores description: [`android/app/src/fdroid/play/`](../android/app/src/fdroid/play/)
-  - Apple App Store description: [`iphone/metadata/`](../iphone/metadata/)
+- **Android:**
 
-  - Popular brands of map features: [`data/strings/brands_strings.txt`](../data/strings/brands_strings.txt)
-  - Search keywords for popular brands: [`data/categories_brands.txt`](../data/categories_brands.txt)
-  - Search keywords for cuisine types: [`data/categories_cuisines.txt`](../data/categories_cuisines.txt)
+  - `android/app/src/main/res/values/strings.xml` – Native Android XML format for UI strings.
 
-  - Country / map region names: [`data/countries_names.txt`](../data/countries_names.txt)
+- **iOS:**
 
-  - [other strings](STRUCTURE.md#strings-and-translations) files
+  - `iphone/Maps/LocalizedStrings/en.lproj/Localizable.strings` – Standard Xcode format for UI strings.
+  - `iphone/Maps/LocalizedStrings/en.lproj/Localizable.stringsdict` – Xcode format for handling plurals.
```

Comment:
`iphone/plist.txt`?

Reply:
Ok, this file is not even in Weblate.

Reply:
https://hosted.weblate.org/projects/organicmaps/ios-plist/